recipes/soxs/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "soxs" %}
+{% set version = "4.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/soxs-{{ version }}.tar.gz
+  sha256: 924a88fa1c5acf7611a0888e86ce6bf7972c9801a26fa3e04a1657eb961a88d2
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - numpy
+    - pip
+    - cython >=0.29.21
+  run:
+    - python
+    - astropy >=3.0
+    - tqdm
+    - pooch
+    - h5py
+    - scipy
+    - pyyaml
+    - regions
+    - appdirs
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - soxs
+  commands:
+    - pip check
```

```suggestion
  commands:
    - pip check
    - get_instrument_data --help
```
Test that your scripts are installed correctly!